clusters/kind-cluster/bob/sqnc-node/values.yaml

```diff
@@ -1,24 +1,19 @@
 node:
   chain: local
   role: validator
-  dataVolumeSize: 5Gi
+  dataVolumeSize: 1Gi
   customNodeKey: "0000000000000000000000000000000000000000000000000000000000000002"
   perNodeServices:
     setPublicAddressToExternal:
       enabled: false
   flags:
     - "--rpc-external"
-    - "--ws-external"
     - "--rpc-methods=Unsafe"
     - "--rpc-cors=all"
-    - "--unsafe-ws-external"
     - "--unsafe-rpc-external"
     - "--bob"
     - "--prometheus-external"
-    - "--bootnodes '/dns4/alice-node-dscp-node-0-rc-p2p.alice.svc.cluster.local/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp'"
-    - "--telemetry-url 'ws://telemetry-shard.dscp.svc.cluster.local/submit 1'"
-  serviceMonitor:
-    enabled: true
-    interval: 10s
-    scrapeTimeout: 10s
+    - "--bootnodes '/dns4/alice-node-sqnc-node-0-rc-p2p.alice.svc.cluster.local/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp'"
 storageClass: standard
+image:
+  tag: v11.0.0
```

Comment:
Any reason we aren't using the version listed in the chart?

Reply:
the chart refers to 11.0.0 so no. Reverted